Makefile

```diff
@@ -40,19 +40,30 @@ black:
 flake8:
 	flake8 $(ALLPY)
 
-MYPY = pyteal scripts tests
 mypy:
-	mypy --show-error-codes $(MYPY)
+	mypy
 
 lint: black flake8 mypy
 
 # ---- Unit Tests (no algod) ---- #
 
 # TODO: add blackbox_test.py to multithreaded tests when following issue has been fixed https://github.com/algorand/pyteal/issues/199
 NUM_PROCS = auto
-test-unit:
-	pytest -n $(NUM_PROCS) --durations=10 -sv pyteal tests/unit --ignore tests/unit/blackbox_test.py --ignore tests/unit/user_guide_test.py
-	pytest -n 1 -sv tests/unit/blackbox_test.py tests/unit/user_guide_test.py
+test-async-unit:
+	pytest -n $(NUM_PROCS) --durations=10 -sv pyteal tests/unit \
+		--ignore tests/unit/blackbox_test.py \
+		--ignore tests/unit/user_guide_test.py \
+		--ignore tests/unit/sourcemap_test.py \
+		--ignore tests/unit/sourcemap_monkey_raises_test.py \
+		--ignore tests/unit/sourcemap_monkey_unit_test.py
+
+test-sync-unit:
+	pytest -n 1 --durations=10 -sv tests/unit/blackbox_test.py tests/unit/user_guide_test.py tests/unit/sourcemap_test.py tests/unit/sourcemap_monkey_raises_test.py
+
+test-sync-unit-monkey:
+	pytest -n 1 --durations=10 -sv tests/unit/sourcemap_monkey_unit_test.py
+
+test-unit: test-async-unit test-sync-unit test-sync-unit-monkey
```

Comment:
TODO: consolidate these commands - it ought to be much simpler now that we have the `-m serial` option